### ISSUE TITLE
Bleedthrough cv2 resize issue

### ISSUE THREAD

**Issue by proofconstruction** [sparkfish/augraphy#219]
Putting this here mostly as a note to my future self.

Sometimes the foreground image fails to resize properly:

```
[/usr/local/lib/python3.8/dist-packages/augraphy/augmentations/bleedthrough.py] in create_bleedthrough_foreground()
    138 
    139             # resize foreground
--> 140             image_bleedthrough_foreground = cv2.resize(
    141                 image_bleedthrough_foreground,
    142                 (image.shape[1], image.shape[0]),

error: OpenCV(4.6.0) /io/opencv/modules/imgproc/src/resize.cpp:4052: error: (-215:Assertion failed) !ssize.empty() in function 'resize'
```

**Comment by kwcckw** [comment]
I think it's either image is empty or the size of image width or height is zero.  Are you able to reproduce this problem consistently?

**Comment by proofconstruction** [comment]
Fixed with [commit 4ca1d51](https://github.com/sparkfish/augraphy/commit/4ca1d51ca7083d38e1b9b1f4b9b4bd30a7e5afc7)
